src/components/Card/Card.styled.jsx

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+
+const CardWrapper = styled.div`
+  border-radius: 4px;
+  box-shadow: 0px 2px 1px -1px rgb(0 0 0 / 20%), 0px 1px 1px 0px rgb(0 0 0 / 14%), 0px 1px 3px 0px rgb(0 0 0 / 12%);
+  height: 345px;
```

Comment:
I suggest you use relative units like rem. This helps you achieve a more responsive application. Ideally, you should only use pixels for font sizes.

https://thecssworkshop.com/lessons/relative-units